### PR TITLE
Created method to reformat with black prior to linting.

### LINKED ISSUE #170: Create method to auto-lint with black
Wrap the existing `make lint` with `make auto-lint` that reformats using black before linting.

### PR DESCRIPTION
**Pull request by TuTomasz**
### Issue

Fixes #170 

### Description
Implements method auto-lint in the MakeFile to perform code reformat using black prior to linting.
Auto-lint wraps the lint method. Introduced it in "make all" as part of the build process. 

### Testing
Code review.

### Checklist
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] 🤔 **CONSIDER** adding a unit test if your PR resolves an issue.
- [x] ✅ **DO** check open PR's to avoid duplicates.
- [x] ✅ **DO** keep pull requests small so they can be easily reviewed.
- [x] ✅ **DO** build locally before pushing.
- [x] ✅ **DO** make sure tests pass.
- [x] ✅ **DO** make sure any new changes are documented.
- [x] ✅ **DO** make sure not to introduce any compiler warnings.
- [x] ❌**AVOID** breaking the continuous integration build.
- [x] ❌**AVOID** making significant changes to the overall architecture.

💚Thank you!

### CODE REVIEW

**Comment by keithrfung** [comment]
Thanks for your pull request! We'll be reviewing shortly.

File: Makefile [hunk 1]
```diff
@@ -8,7 +8,7 @@ IS_64_BIT ?= $(shell python -c 'from sys import maxsize; print(maxsize > 2**32)'
 SAMPLE_BALLOT_COUNT ?= 5
 SAMPLE_BALLOT_SPOIL_RATE ?= 50
 
-all: environment install validate lint coverage
+all: environment install validate auto-lint coverage
```
Comment (rkorsak) [inline Makefile:11]: @keithrfung / @AddressXception - any objection to running the auto-lint here?

**Review by rkorsak — COMMENTED** [review]
I've confirmed this works exactly as expected locally, and I've wanted exactly this feature all day today 😆.

Thanks for the contribution!

I have one small open question to other folks, but otherwise this looks good to me.

**Comment by TuTomasz** [comment]
Awesome no problem glad to be of help 😃.

File: Makefile (hunk 1 above)
Comment (keithrfung) [reply Makefile:11]: No. That's good for my perspective. Makes sense.

**Review by rkorsak — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (AddressXception) [reply Makefile:11]: im in favor too

**Review by AddressXception — APPROVED** [review]

**Pull request merged**